php/commands/media.php

```diff
@@ -233,14 +233,14 @@ private function remove_old_images( $att_id ) {
 		$wud = wp_upload_dir();
 
 		$metadata = wp_get_attachment_metadata( $att_id );
+		
+		if ( !is_array($metadata) || !array_key_exists('sizes', $metadata) || empty($metadata['sizes'])) {
+			return;
+		}
 
 		$dir_path = $wud['basedir'] . '/' . dirname( $metadata['file'] ) . '/';
 		$original_path = $dir_path . basename( $metadata['file'] );
 
-		if ( empty( $metadata['sizes'] ) ) {
```

Comment:
The check is still there, it just goes above creating of the paths as they are not needed when it fails.